torch/csrc/jit/codegen/cuda/executor.h

```diff
@@ -73,6 +73,10 @@ class TORCH_CUDA_API FusionExecutor : public NonCopyable {
     uint64_t rand_offset;
   };
 
+  KernelSummary summary() const {
```

Comment:
Let's expose the kernel object instead, and we can say `fe->kernel()->summary()'

In general, this kind of "shallow" interfaces are not recommended (for multiple reasons, one being that they make the parent interface - the FusionExecutor in this case, more complex for no good reason. We can discuss this separately if you'd like)